src/routes.ts

```diff
@@ -182,4 +195,37 @@ routes.get("/", async (_req, res) => {
   });
 });
```

Comment:
Feel free to delete `/` if it's getting in the way, don't think any of it is needed anymore (especially since all that nonZeroBalance stuff isn't needed anymore)